plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -35,6 +37,10 @@
     public static final String FILE_FORMAT_PROPERTY = "format";
     public static final String PARTITIONING_PROPERTY = "partitioning";
     public static final String LOCATION_PROPERTY = "location";
+    public static final String SPLIT_SIZE_PROPERTY = "read.split.target-size";
```

Comment:
I'm not sure we necessarily need to be able to get/set the values as TableProperties. It might be enough to have them set as config/session properties. 

I see that in Iceberg it is set on the table level but specifying a split size at table creation doesn't seem right to me for Trino. 

I also think it's likely that Spark and Trino would benefit most from different split size in some cases so being able to use the Trino config rather than the TableProperty might be nice.

Reply:
Yes, I also doubt whether it's a good idea to set these properties on table or not at first, but considering Spark/Hive can also read the same table created & written by Trino and read these table properties to plan tasks by default, I add this logic, of course Spark can also set split size on session.

In our practices, we will set split size as table properties though Spark, so that all the Trino queries can be effected easily and coarsely.

All in all, could you give me a decision if we should remove the logic of setting split size as table properties?